### PR TITLE
Indicator size calculation should care about the ProgressBar's Padding property setting

### PR DESCRIPTION
**Pull request by comesx4**
## What does the pull request do?
Make the ProgressBar Indicator size calculation correct with Padding setted.

## What is the current behavior?
If Padding setted, the Indicator will overflow while value is 100%.

image

## What is the updated/expected behavior with this PR?
The Indicator's size calculation is affected by Padding property.

image

## How was the solution implemented (if it's not obvious)?
Minus Indicator Margin left and right or top bottom while calculating size.

## Checklist

~~- [ ] Added unit tests (if possible)?~~
- [x ] Added XML documentation to any related classes?
~~- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation~~

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0025885-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by comesx4** [comment]
How about this PR, is there any problem with it? if any, I'd like to fix it

**Comment by timunie** [comment]
I think the team is just busy atm. We appreciate your work and you will get a feedback if anything is wrong. Otherwise I think your PR will be merged soon.

File: src/Avalonia.Controls/ProgressBar.cs [hunk 1]
```diff
@@ -270,15 +270,16 @@ private void UpdateIndicator()
                     double percent = Maximum == Minimum ? 1.0 : (Value - Minimum) / (Maximum - Minimum);
 
                     // When the Orientation changed, the indicator's Width or Height should set to double.NaN.
+                    // Indicator size calculation should consider the ProgressBar's Padding property setting
```
Comment (maxkatz6) [inline src/Avalonia.Controls/ProgressBar.cs:273]: Comments says about "Padding" property, but below you substract by Margin property.

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0027023-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-nightly/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**